torch/csrc/jit/passes/shape_analysis.cpp

```diff
@@ -540,10 +553,17 @@ class ShapePropagator {
         return;
       }
       case aten::_unwrap_optional: {
-        auto input_ivalue = toIValue(node->input());
-        if (input_ivalue && input_ivalue->isNone()) {
+        // During analysis we don't want to pass None through here
```

Comment:
same here

Reply:
Again, it might be element (sub-) type after specialization.